e2e_local.ps1

```diff
@@ -0,0 +1,25 @@
+$ErrorActionPreference="stop" 
+
+if ($Args.Count -eq 0) {
```

Comment:
A more powershell idiomatic way to deal with this would be to put this at the top of the script:
```
Param(
   [Parameter(Mandatory=$true)]
   [string[]]$TestArgs
)
```
If the user dose not supply the args, they will be prompted:

```
C:\dev\habitat [named_pipe]> .\e2e_local.ps1                                                                                                                                                                                                         cmdlet test.ps1 at command pipeline position 1
Supply values for the following parameters:
TestArgs[0]: blah
TestArgs[1]:
```